wouso/core/magic/tests.py

```diff
@@ -179,23 +179,35 @@ def test_dispell_no_due(self):
         player.magic.cast_spell(dispell, player)
         self.assertEqual(PlayerSpellDue.objects.filter(player=player).__len__(), 0)
 
-    def test_cure(self):
+    def test_cure_negative(self):
         """
-         Test if cure works on a player
+         Test if cure works on a negative spell
         """
         player = self._get_player()
-        player2 = self._get_player(2)
 
         spell = Spell.objects.create(name='test-spell', available=True, price=10, type='n')
         cure = Spell.objects.create(name='cure', available=True, price=10)
         obs = PlayerSpellDue.objects.create(player=player, source=player, spell=spell, due=datetime.now() + timedelta(days=1))
 
-        self.assertTrue(player.magic.spells) # There is test-spell cast on myself
+        player.magic.add_spell(cure)
+        player.magic.cast_spell(cure, player, datetime.now() + timedelta(days=1))
+
+        self.assertEqual(len(PlayerSpellDue.objects.filter(player=player)), 0) # There isn't any spell left
```

Comment:
@iulianR Use QuerySet.count() instead of len(QuerySet). From django docs:

Note: Don't use len() on QuerySets if all you want to do is determine the number of records in the set. It's much more efficient to handle a count at the database level, using SQL's SELECT COUNT(*), and Django provides a count() method for precisely this reason.